params/safety_config.yaml

```diff
@@ -0,0 +1,11 @@
+safety_bonds:
```

Comment:
But it isn't even a matter of loading included modules first, it's that within those included modules you need to have base_hardware first